website/docs/terminology.md

```diff
@@ -142,11 +179,31 @@ See also: [Prover documentation], [Executor]
 ### Receipt
 
 [receipt]: #receipt
+[receipts]: #receipt
 
-A receipt attests to valid [execution] of a [guest program]. [Verifying] the receipt provides cryptographic assurance that the [journal] was indeed constructed using the expected [circuit] and the expected [imageID].
+A receipt attests to valid [execution] of a [guest program].
+[Verifying] the receipt provides cryptographic assurance that the [journal] was indeed constructed using the expected [circuit] and the expected [imageID].
 The receipt consists of a [journal] and a [seal].
```

Comment:
Do we want to update this to point at `[receipt claim]` instead of `[journal]`? I'm a bit ambivalent but it seems like "journal + seal" is not quite what a receipt is these days...

Reply:
Good suggestion. I've addressed this in 049002808dad918709891283c55521f07000f405